app/views/routes/edit.html.erb

```diff
@@ -0,0 +1,27 @@
+<h1>Edit Route</h1>
```

Comment:
Are you really create it manually without scaffold ? Or it's copy pasta ?

Reply:
It's copypasta just as in lesson